int16 id
float64 longitude
float64 latitude
float64 s
float64 x
float64 y
float64 theta 
int16 attribute
